server/mhn/templates/base.html

```diff
@@ -48,10 +48,14 @@
 			 <li class="has-dropdown">
                         <a href="/ui/chart">Charts</a>
                         <ul class="dropdown">
-                            <li><a href="/image/top_passwords.svg">Kippo Top Passwords</a></li>
```

Comment:
Why not just combine all the user/password analytics together?  

ex: Kippo/Cowrie Top Users
etc

I image most people will run either Kippo or Cowrie and very few would run both or have need to see these separated.